src/Castle.Core.Tests/ProxyGenerationOptionsTestCase.cs

```diff
@@ -23,7 +24,9 @@ namespace Castle.DynamicProxy.Tests
 
 	using NUnit.Framework;
 
-	[TestFixture]
+	using DescriptionAttribute = System.ComponentModel.DescriptionAttribute;
+
+    [TestFixture]

```

Comment:
@cmerat, there's an indentation problem here. Castle.Core uses tabs for indentation, not spaces.